README.md

```diff
@@ -96,6 +98,11 @@ If existing code gets reformatted, this must be done in a separate commit. Its c
 ### Running CI tests locally
 To avoid unneeded load on the GitHub infrastructure, please consider running `tools/ci/run_ci.sh --all` before pushing.
 
+### Running integration tests locally
+`cd wakaama`
+`tools/ci/run_ci.sh --run-build`
+`pytest -v tests/integration`
```

Comment:
This looks bad when being rendered:
![image](https://user-images.githubusercontent.com/920644/119488568-fe888c00-bd5a-11eb-9395-d9b552396c3f.png)

Propsal:

```suggestion
```bash
cd wakaama
tools/ci/run_ci.sh --run-build
pytest -v tests/integration
```
```
(you can leave out the "bash", had to add it as otherwise the suggestions functionality breaks.

Reply:
Good catch. I added tripple backticks on the line above and below and it renders nicely on my machine...

Reply:
added pylint version printout to commit message